brakeman.log

```diff
@@ -0,0 +1,25 @@
+
++BRAKEMAN REPORT+
```

Comment:
I don't think you want this checked in to source control. This should be in gitignore